kolibri/core/analytics/tasks.py

```diff
@@ -70,3 +71,11 @@ def schedule_ping(
             pass
     elif conf.OPTIONS["Deployment"]["DISABLE_PING"]:
         job_storage.clear(job_id=DEFAULT_PING_JOB_ID)
+
+
+def trigger_pingback():
```

Comment:
I think it may be sufficient to just do `schedule_ping` rather than trying to be absolutely sure that we manage to do a pingback after every single invocation. This will still increase the density of pingback events and should give us better data.